src/MOI_wrapper.jl

```diff
@@ -216,4 +244,155 @@ function backward!(model::Optimizer, params::Array{String}, dl_dz::Array{Float64
+    MOI.set(model.optimizer, MOI.VariablePrimalStart(), vi, value)
+end
+
+function MOI.supports(model::Optimizer, ::MOI.VariablePrimalStart,
+                      ::Type{MOI.VariableIndex})
+    return MOI.supports(model.optimizer, MOI.VariablePrimalStart(), MOI.VariableIndex)
+end
+
+function MOI.get(model::Optimizer, attr::MOI.VariablePrimal, vi::MOI.VariableIndex)
+    MOI.check_result_index_bounds(model.optimizer, attr)
+    return MOI.get(model.optimizer, attr, vi)
+end
+
+function MOI.add_constraint(model::Optimizer, f::MOI.SingleVariable, s::SUPPORTED_SCALAR_SETS)
+    ci = MOI.add_constraint(model.optimizer, f, s)
+    push!(model.con_idx, ci)
+    return ci
+end
+
+function MOI.delete(model::Optimizer, ci::CI{F,S}) where {F <: SUPPORTED_SCALAR_FUNCTIONS, S <: SUPPORTED_SCALAR_SETS}
+    filter!(e -> e≠ci, model.con_idx)
+    MOI.delete(model.optimizer, ci) 
+end
+
+function MOI.get(model::Optimizer, ::MOI.ConstraintPrimal, ci::CI{F,S}) where {F <: SUPPORTED_SCALAR_FUNCTIONS, S <: SUPPORTED_SCALAR_SETS}
+    return MOI.get(model.optimizer, MOI.ConstraintPrimal(), ci)
+end
+
+function MOI.is_valid(model::Optimizer, v::VI)
+    return v in model.var_idx
+end
+
+function MOI.is_valid(model::Optimizer, con::CI)
+    return con in model.con_idx
+end
+
+function MOI.get(model::Optimizer, ::MOI.ConstraintDual, ci::CI{F,S}) where {F <: SUPPORTED_SCALAR_FUNCTIONS, S <: SUPPORTED_SCALAR_SETS}
+    return MOI.get(model.optimizer, MOI.ConstraintDual(), ci)
+end
+
+function MOI.get(model::Optimizer, ::MOI.ConstraintBasisStatus, ci::CI{F,S}) where {F <: SUPPORTED_SCALAR_FUNCTIONS, S <: SUPPORTED_SCALAR_SETS}
+    return MOI.get(model.optimizer, MOI.ConstraintBasisStatus(), ci)
+end
+
+# helper method to check if a constraint contains a Variable
+function _constraint_contains(model::Optimizer, v::VI, ci::CI{F,S}) where {F <: SUPPORTED_SCALAR_FUNCTIONS, S <: SUPPORTED_SCALAR_SETS}
+    func = MOI.get(model, MOI.ConstraintFunction(), ci)
+    if func isa MOI.SingleVariable
+        return v == func.variable
+    elseif func isa MOI.ScalarAffineFunction{Float64}
+        for term in func.terms
+            if term.variable_index == v
+                return true
+            end
+        end
+        return false
+    end
+    return false   # default
+end
+
+function MOI.delete(model::Optimizer, v::VI)
+    # remove those constraints that depend on this Variable
+    filter!(ci -> !_constraint_contains(model, v, ci), model.con_idx)
+
+    # delete in inner solver 
+    MOI.delete(model.optimizer, v) 
+
+    # delete from var_idx
+    filter!(e -> e ≠ v, model.var_idx)
+end
+
+# for array deletion
+function MOI.delete(model::Optimizer, indices::Vector{VI})
+    for i in indices
+        MOI.delete(model, i)
+    end
+end
+
+function MOI.modify(
+    model::Optimizer,
+    ::MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}},
+    chg::MOI.AbstractFunctionModification
+)
+    MOI.modify(model.optimizer, 
```

Comment:
```suggestion
    MOI.modify(
        model.optimizer, 
```

Reply:
and allign the next arguments with the first, with the closing parenthesis on a new line

Reply:
fixed